src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.navigation {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: flex-end;
+  }
+
+  &__item {
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    height: 60px;
+    line-height: 60px;
+    text-decoration: none;
+    color: $nav-link-color;
+
+    &.is-active {
+      color: $nav-link-hover-color;
+      position: relative;
+    }
+
+    &.is-active::after {
+      content: '';
+      width: 37px;
+      height: 4px;
+      background-color: $nav-link-hover-color;
```

Comment:
The background color for the active link indicator is using the same variable as the hover color. If the design specifies different colors for hover and active states, make sure to use the appropriate variables for each state.